.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: ruby
+rvm:
+  - "1.9.3"
```

Comment:
I pushed a change to gemspec limiting ruby to 2.0 and up, and released a new gem version 0.1.2 to include that change as well all watches/rubocop/events changes